packages/indexer/indexer/chain_mapper.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+from typing import Dict, List
+
+
+@dataclass
+class CosmosChain:
+    chain_id: str
+    min_block_height: int
+    apis: List["CosmosAPI"] = field(default_factory=list)
+    current_api_index: int = 0
+
+
+@dataclass
+class CosmosAPI:
+    url: str
+    archive: bool = False
+
+
+chain_mapping: List[CosmosChain] = [
+    CosmosChain(
+        chain_id="secret-4",
+        min_block_height=7284419,
```

Comment:
I don't like that these are hardcoded. What's the plan for when we move to `secret-5`?

Reply:
well we will have to change api endpoints for secret-5 anyways..its a different chain

Reply:
Right, hence they should be env variables